### DISCUSSION TITLE
Please remove (or stub) types for [glob]

### DISCUSSION THREAD

**Discussion by isaacs** [DefinitelyTyped/DefinitelyTyped#65038] [Issues with a @types package]
It's causing issues for users of rimraf.  These libraries are now written in typescript and ship types of their own.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/glob/index.d.ts

https://github.com/isaacs/rimraf/issues/264

https://stackoverflow.com/questions/75890950/node-modules-minimatch-dist-cjs-index-has-no-exported-member-ioptions

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
fyi: there is ongoing effort to remove @types/glob #64810

**Reply by isaacs** [reply]
If anyone can help that process along, it'd be much appreciated by users of glob that are currently running into issues with it.

It's failing for some seemingly unrelated issue, where the `inquirer` type is not being built prior to being pulled in by the linter, so it fails to find `rxjs`.  None of which seems related to glob at all.
